p4-16/spec/P4-16-spec.mdk

```diff
@@ -6973,7 +6973,7 @@ The P4 compiler should provide:
 |-----|-----|-----|
 | 1.0.0 | May 17, 2017 | Initial version. |
 |-----|-----|-----|
-| 1.1.0 | June 5, 2018 | Added top-level functions, optional and named parameters, `enum` |
+| 1.1.0 | November 26, 2018 | Added top-level functions, optional and named parameters, `enum` |
 |       |          | representations, parser value sets, type definitions, saturating |
 |       |          | saturating arithmetic, and structured annotations. |
```

Comment:
"saturating" is repeated

Reply:
You mentioned "rolling thunder" ... we are going for as many saturatings as possible :-)